splice/static/js/controllers/distribution.js

```diff
@@ -107,6 +117,14 @@ angular.module('spliceApp').controller('distributionController', function($scope
 
       for (var i = 0; i < tiles.length; i++) {
         var tile = tiles[i];
+
+        // populate the imageURI and enhancedImageURI if tile is in compact format
+        if (tile.hasOwnProperty("imageURI") && assets.hasOwnProperty(tile.imageURI)) {
+          tile.imageURI = assets[tile.imageURI];
+        }
+        if (tile.hasOwnProperty("enhancedImageURI") && assets.hasOwnProperty(tile.enhancedImageURI)) {
+          tile.enhancedImageURI = assets[tile.enhancedImageURI];
+        }
```

Comment:
@oyiptong on line 151 below, chanId seems to be an object, and "chanId + url" turns out be "[object Object]https:aws....com". I wonder did you mean "var chanId = $scope.channelSelect.id"?